packages/util/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 8.0.6 - 2023-04-20
+
+- Bump `@chainsafe/ssz` dependency to 0.11.0 (no WASM, native SHA-256 implementation, ES2019 compatible), PRs [#2622](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2622) and PR [#2564](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2564)
```

Comment:
I would be curious to ask some folks who complained about the WASM dependency if this does indeed resolve their issue (switching to `noble-curves` still appears to install `as-sha256` - at least for us since I had to manually patch that in the karma test runner config)

Reply:
I opened a PR that seems to fix this here: https://github.com/ChainSafe/ssz/pull/318 - pending review